x/supplier/keeper/query_claim_test.go

```diff
@@ -46,37 +48,58 @@ func TestClaim_QuerySingle(t *testing.T) {
 				SupplierAddress: claims[1].SupplierAddress,
 			},
 
-			response: &types.QueryGetClaimResponse{Claim: claims[1]},
-			err:      nil,
+			response:    &types.QueryGetClaimResponse{Claim: claims[1]},
+			expectedErr: nil,
 		},
 		{
 			desc: "Claim Not Found - Random SessionId",
 
 			request: &types.QueryGetClaimRequest{
 				SessionId:       "not a real session id",
-				SupplierAddress: claims[0].SupplierAddress,
+				SupplierAddress: claims[0].GetSupplierAddress(),
 			},
 
-			err: status.Error(codes.NotFound, "claim not found"),
+			expectedErr: status.Error(
+				codes.NotFound,
+				types.ErrSupplierClaimNotFound.Wrapf(
+					// TODO_CONSIDERATION: factor out error message format strings to constants.
```

Comment:
I feel like having them inline (with the wrapper) is the right amount of abstraction: centralized error type with inline string. If we move out the strings into constants, it might be too much jumping around.

Lmk wyt so we can decide on keeping the TODOs in the PR or not.

Reply:
I'm gonna revert the TODO, thanks for the perspective.